taskmanager/features/report.feature

```diff
@@ -0,0 +1,29 @@
+# language: en
+Feature: Generate a report
+  In order to export tasks with details
+  As a user or system
+  I want to generate various of reports
+
+  Scenario: Generate report for TODO tasks
+    Given There is no report
+    And there are "2" tasks with "TODO" status
+    When I generate report for "TODO" tasks
```

Comment:
How does it work? Any more detailed reqs for reports?
IMHO reports should present data like: 
- unfinished
- done 
- inprogress
- some charts and comparision between them

what was your a first idea when you said "Reports feature"? Could you explain to us that bit more?